src/tests/components/ActivitiesListFilter.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import ActivitiesListFilter from '../../components/ActivitiesListFilter';
```

Comment:
I would make a big suggestion here. 
It would be much more cleaner if you put your test files next to the components.
Do it in next PR, but starting from this next PR, you will have separate folder for *each* component.
Like:
`components/ActivitiesListFilter/ActivitiesListFilter.js`
`components/ActivitiesListFilter/ActivitiesListFilter.test.js`
the you will be importing from `...ActivitiesListFilter/ActivitiesListFilter`. 
If you wish, you can also add index.js file to this folder with line:
`export { default as ActivitiesListFilter }` to import like `import ActivitiesList from "...ActivitiesListFilter". 

Remember, next PR